src/components/Slider/Slider.module.css

```diff
@@ -143,7 +143,7 @@
  */
 .slider__markers {
   display: flex;
-  align-items: center;
+  align-items: top;
```

Comment:
Not sure if `top` is a valid [keyword](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items) for `align-items`

Reply:
good catch, it is not. will update